pkg/apis/openebs.io/v1alpha1/cas_template_keys.go

```diff
@@ -41,6 +41,12 @@ const (
 	// top level property
 	ConfigTLP TopLevelProperty = "Config"
 
+	// RuntimeTLP is a top level property supported by CAS template engine
+	//
+	// The policy specific properties are placed with RuntimeTLP as the
+	// top level property
+	RuntimeTLP TopLevelProperty = "Runtime"
```

Comment:
Can we avoid this.
upgrade engine code should take care of this.

Reply:
removed